src/js/tabs/trade.controller.js

```diff
@@ -446,6 +448,29 @@ TradeTab.prototype.angular = function(module)
       }
     };
 
+
+    /**
+     * Returns orders currency pair, so we can compare it with current pair.
+     */
+    function getOrderCurrency(entry) {
+      if (!entry) return '';
+      var first_currency = entry.first.currency().to_json();
+      var first_issuer = entry.first.issuer().to_json();
+      var second_currency = entry.second.currency().to_json();
+      var second_issuer = entry.second.issuer().to_json();
+
+      var first = first_currency == 'XRP'
```

Comment:
Is this a non-strict comparison by reason (maybe because `entry.first.currency().to_json()` may not be a string)?